### PR TITLE
std.parallelism: move static constructor out of template

### PR DESCRIPTION
**Pull request by WalterBright**
The original had it inside a template, when the ctor has no dependencies on the template type. Also avoids problems with duplicate copies of the template instantiation, including 0.

### CODE REVIEW

**Comment by WalterBright** [comment]
Also removed redundant declaration of GetSystemInfo() and both redundant and wrong declaration of SYSTEM_INFO.

File: std/parallelism.d [hunk 1]
```diff
@@ -103,35 +102,10 @@ else version(FreeBSD)
 version(Windows)
 {
     // BUGS:  Only works on Windows 2000 and above.
```
Comment (DmitryOlshansky) [inline std/parallelism.d]: We don't support anything below. Not a bug then but just a note.

**Comment by DmitryOlshansky** [comment]
LGTM

**Comment by schveiguy** [comment]
Auto-merge toggled on

**Pull request merged**